src/opera/parser/tosca/v_1_3/requirement_definition.py

```diff
@@ -22,3 +22,9 @@ def normalize(cls, yaml_node):
         if isinstance(yaml_node.value, str):
             return Node({Node("capability"): yaml_node})
         return yaml_node
+
+    @classmethod
+    def validate(cls, yaml_node):
+        super().validate(yaml_node)
+        if "relationship" in yaml_node.bare and not isinstance(yaml_node.bare["relationship"], str):
+            cls.abort("Expected a relationship string reference.", yaml_node.loc)
```

Comment:
Maybe we can be more specific here: "Expected a relationship type name as a 'relationship' value."

Reply:
Ok, I chaged that as you suggested.